app/ui/legacy/src/main/java/com/fsck/k9/activity/MessageListActivityAppearance.kt

```diff
@@ -0,0 +1,67 @@
+package com.fsck.k9.activity
+
+import com.fsck.k9.Account
+import com.fsck.k9.FontSizes
+import com.fsck.k9.K9
+
+class MessageListActivityAppearance(accounts: List<Account>) {
+    val appTheme = K9.appTheme
+    val k9Language = K9.k9Language
+    val isHideSpecialAccounts = K9.isHideSpecialAccounts
+    val isShowMessageListStars = K9.isShowMessageListStars
+    val isShowCorrespondentNames = K9.isShowCorrespondentNames
+    val isMessageListSenderAboveSubject = K9.isMessageListSenderAboveSubject
+    val isShowContactName = K9.isShowContactName
+    val isChangeContactNameColor = K9.isChangeContactNameColor
+    val isShowContactPicture = K9.isShowContactPicture
+    val isColorizeMissingContactPictures = K9.isColorizeMissingContactPictures
+    val isUseBackgroundAsUnreadIndicator = K9.isUseBackgroundAsUnreadIndicator
+    val contactNameColor = K9.contactNameColor
+    val messageViewTheme = K9.messageViewTheme
+    val messageListPreviewLines = K9.messageListPreviewLines
+    val splitViewMode = K9.splitViewMode
+    val fontSizes = K9.fontSizes
+    val chipColor = accounts.map { it.chipColor }
```

Comment:
There's a lot more that can change with accounts. So we need to handle this differently anyway. For now I suggest to just leave out the chipColor and concentrate on general display settings.